apps/backend/src/tests/neighborhood_api.test.ts

```diff
@@ -153,7 +90,7 @@ describe('When neighborhoods already exist in the db', () => {
     expect(response.body.error).toEqual('No Neighborhood found');
   });
 
-  test('GET /neighborhoods/id only returns the id, name, description and location of neighborhood if user is not logged in', async () => {
+  test('GET /neighborhoods/:id only returns the id, name, description and location of neighborhood if user is not logged in', async () => {
```

Comment:
Can you bring this up at our next meeting? I think this is the behavior we've decided on.

Reply:
My understanding was that guest users would only see the all neighborhoods display, with a log in/sign up thing. My understanding was that the `GET /neighborhoods/:id` route was only accessible to logged-in users.

I'd like to suggest that we document features that we have decided on, and consider features that aren't documented as undecided. If we have a document that we all understand to be the "source of truth" for the features we're implementing, it will be easier to implement features.

Reply:
Right now guest and non-member users are getting the same info about a neighborhood. We might want to change that.